src/main/java/com/kodilla/ecommercee/dto/CartDto.java

```diff
@@ -0,0 +1,16 @@
+package com.kodilla.ecommercee.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class CartDto {
+    private Long cartId;
+    private Long userId;
```

Comment:
Check cart json specification for this.
I think we need to discuss translating our ERD into classes in the context of the json specification here at all because this is the first place where it totally breaks down... :/